s32160-pj-Pawel_Rohde/udostepnienia_dostepu_cwiczenia.txt

```diff
@@ -0,0 +1,236 @@
+
+*s32160@szuflandia22:~lab01$ cd -
+/home/PJWSTK/s32160
+s32160@szuflandia22:~$
+Przy pomocy polecenia 'cd -' moge wrocic do katalogu domowego, wczeniej znajdujac sie w lab02, ale nie majac uprawnien dostepu do
+katalogu ukos.
+Natomiast wrocic przy pomocy polecenia 'cd -' do katalogu lab02 już nie moge bo nie mam uprawnien wstepu do ukos.
+
+3.W zespolach 2- lub 3-osobowych (w opisie zadania na githubie prosze umiescic login osob z zespolu) wyprobowac mozliwosc komunikacji przez wspoldzielony
+plik: na jednym z kont w zespole utworzyc pusty plik i przydzielic odpowiednie uprawnienia dostepu (do pliku i do katalogu domowego). Wpisywac i odczytywac komunikaty przy uzyciu polecen:
+
+echo tresc_komunikatu > plik
+
+cat plik
+
+Sprawdzic, jaki skutek powoduje zamiana operatora > na operator >> w poleceniu echo. Uruchom takze drugi terminal i wykonaj w nim komende tail -f plik i powtorz powyzsze cwiczenie w pierwszym terminalu (komunikacja za pomoca pliku).
+
+(Zadanie wykonywane z s30397)
+
+*s30899@szuflandia22:~/lab01$ touch komunikacja
+*s30899@szuflandia22:~/lab01$ chmod g+w komunikacja
+*s30899@szuflandia22:~/lab01$ ls -l
+-rw-rw-r-- 1 s30899 domain users    0 10-27 02:07 komunikacja
+
+*s30899@szuflandia22:~/lab01$ echo siema > komunikacja
+*s30899@szuflandia22:~/lab01$ cat komunikacja
+siema
+
+Po uzyciu komedy echo przez kolege:
+*s30899@szuflandia22:~/ukos/lab02$ cat komunikacja
+jajuszko
+
+Uzywajac komendy 'cat' i 'echo' na zmiane z kolega mozemy komunikować sie przy uzyciu pliku.
+
+*s30899@szuflandia22:~/ukos/lab02$ echo zielone >> komunikacja
+*s30899@szuflandia22:~/ukos/lab02$ cat komunikacja
+jajuszko
+zielone
+
+Zmiana operatora '>' na operator '>>' w poleceniu 'echo' spowodowala, ze nowa tresc zostala dolaczona
+do istniejacej tresci w pliku zamiast ja nadpisywac.
+
+*s30899@szuflandia22:~/ukos/lab02$ tail -f plik
+
+Uruchomienie tej komendy w drugim terminalu pozwolilo na sledzenie zmian w pliku na zywo, gdy uzywalismy komend 'echo' i 'cat'.
+
+4.Znalezc w swoim katalogu domowym podkatalog public_html (jesli go nie ma, to utworzyc; musi sie on nazywac DOKLADNIE tak jak
+podano, pisane malymi literami z podkresleniem zamiast spacji pomiedzy slowami). Umiescic w nim plik o nazwie strona.html o
+nastepujacej zawartosci:
+
+<HTML>
+<BODY>
+<H1>To jest moja strona domowa</H1>
+</BODY>
+</HTML>
+
+Sprawdzic, jakie sa minimalne uprawnienia dostepu, ktore trzeba przydzielic do:
+
+-katalogu domowego;
+
+-katalogu public_html;
+
+-pliku strona.html, aby zawartosc pliku byla widoczna w przegladarce internetowej
+pod adresem http://szuflandia.pjwstk.edu.pl/~twoj_login/strona.html
+
+*s32160@szuflandia22:~/public_html$ chmod 701 ~
+Minimalne uprawnienia dostepu dla katalogu domowego aby wyswietlic zawartosc pliku w przegladarce internetowej to uprawnienia do
+wejscia dla wszystkich, poniewaz zabierajac uprawnienina dostepu dla wlasciciela do katalogu domowego pewnie tez zadziala, ale nie
+chce tego probowac.
```

Comment:
Ciekawi mnie tylko skąd wiesz, że takie są minimalne wymagania skoro tego nie sprawdziłeś?
Wiedza nie sprawdzona w praktyce to tak jak obietnice szybkiej gotówki w spamie.
Wszędzie jest o tym pełno informacji ale w praktyce nikt nie widział.

Tu co prawda masz rację ale skąd ta wiedza skoro nie jest poprarta samodzielnymi testami?